### PR TITLE
fix(toolbox) don't mix web and native actions on the same file

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: react/features/toolbox/actions.any.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+ * Shows/hides the overflow menu.
+ *
+ * @param {boolean} visible - True to show it or false to hide it.
+ * @returns {{
+ *     type: SET_OVERFLOW_MENU_VISIBLE,
+ *     visible: boolean
+ * }}
+ */
+export function setOverflowMenuVisible(visible: boolean): Object {
+    return {
```
Comment (horymury) [inline react/features/toolbox/actions.any.js:39]: this seems to be web only

File: react/features/toolbox/actions.any.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+/**
+ * Signals that toolbar is hovered value should be changed.
+ *
+ * @param {boolean} hovered - Flag showing whether toolbar is hovered.
+ * @returns {{
+ *     type: SET_TOOLBAR_HOVERED,
+ *     hovered: boolean
+ * }}
+ */
+export function setToolbarHovered(hovered: boolean): Object {
```
Comment (horymury) [inline react/features/toolbox/actions.any.js:54]: web only

File: react/features/toolbox/actions.any.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+
+/**
+ * Signals that toolbox timeout should be cleared.
+ *
+ * @returns {{
+ *     type: CLEAR_TOOLBOX_TIMEOUT
+ * }}
+ */
+export function clearToolboxTimeout(): Object {
```
Comment (horymury) [inline react/features/toolbox/actions.any.js:23]: web only

File: react/features/toolbox/actions.any.js [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+ *
+ * @param {Function} handler - Function to be invoked after the timeout.
+ * @param {number} timeoutMS - Delay.
+ * @returns {{
+ *     type: SET_TOOLBOX_TIMEOUT,
+ *     handler: Function,
+ *     timeoutMS: number
+ * }}
+ */
+export function setToolboxTimeout(handler: Function, timeoutMS: number)
```
Comment (horymury) [inline react/features/toolbox/actions.any.js:106]: web only

File: react/features/toolbox/actions.any.js [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+/**
+ * Dispatches an action which sets new toolbox timeout value.
+ *
+ * @param {number} timeoutMS - Delay.
+ * @returns {{
+ *     type: SET_TOOLBOX_TIMEOUT_MS,
+ *     timeoutMS: number
+ * }}
+ */
+export function setToolboxTimeoutMS(timeoutMS: number): Object {
```
Comment (horymury) [inline react/features/toolbox/actions.any.js:126]: web only

**Review by horymury — CHANGES_REQUESTED** [review]

**Review by horymury — APPROVED** [review]

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by saghul** [comment]
Jenkins please test this please.

**Pull request merged**